WickedEngine/shaders/lightingHF.hlsli

```diff
@@ -42,79 +43,6 @@ inline void ApplyLighting(in Surface surface, in Lighting lighting, inout float4
 	color.rgb += surface.emissiveColor;
 }
 
-inline float3 sample_shadow(float2 uv, float cmp)
```

Comment:
Is it possible to keep the shadow code here instead of moving to a new file? If yes, I would prefer to keep it, because some people already reference shadow code from this file and it will be just confusing if it's suddenly moved. Also, url links to it will be no longer good. If there is a hard dependency issue, then it can be moved to a new file, but if it's only a refactor, then I don't want it.

Reply:
I tried to avoid creating a new file, but since we call shadow in skyAtmosphere.hlsli that also is included in lightingHF.hlsli as well as other files that needed transmittance from atmosphere. Do you have any suggestions what we can do?

Reply:
Ok, if this is dependency problem then it is fine. It wasn't clear to me by looking at changelog.